csrc/device_lower/pass/insert_syncs.cpp

```diff
@@ -1302,17 +1303,29 @@ class WarAsyncWaitInserter : private kir::ExprMutator {
       for (auto it = async_exprs_to_protect_.begin();
            it != async_exprs_to_protect_.end();) {
         auto expr = *it;
+        AsyncOpType type = ir_utils::getAsyncOpType(expr);
+
+        // The wgmma.commit must always be inserted at the end of epilogue for
```

Comment:
Why?

Reply:
The compiler will insert `wgmma.commit` where it thinks it is needed for performance and correctness. 

NvFuser is adding the `wgmma.commit` at the wrong spot, so the kernel get an extra implicit `wgmma.commit`. `ptxas_verbose` indicates a `wgmma.arrive` is added to match this extra `wgmma.commit`.